src/ApiPort/CommandLine/ApiPortConfiguration.cs

```diff
@@ -80,11 +81,12 @@ private static string ArrayTransform(string arg, ConfigMapCount array)
         {
             foreach (var tag in array.Tags)
             {
-                var newKey = $"{array.ExpectedTag}:{array.Count++}";
+                var newKey = string.Format(CultureInfo.InvariantCulture, "{0}:{1}", array.ExpectedTag, array.Count++);
```

Comment:
I had tried this before but unfortunately `FormattableString` is only exposed .NET 4.6, and since our solution targets 4.5, it didn't work. :(